docs/install/install-node.rst

```diff
@@ -6,6 +6,8 @@ Installing the Flocker Node Services
 
 The following instructions describe how to install the ``clusterhq-flocker node`` package, and the optional ``clusterhq-flocker-docker-plugin`` package on each of the nodes in your cluster.
 
+Installing the :ref:`Flocker plugin for Docker <docker-plugin>` at the same time as the other node services is not essential, but it will save you from revisiting each of the nodes in your cluster again if you were to install it at a later time.
```

Comment:
Why isn't it essential?

I think this addition is trivially true.  I'm not sure it is useful, though.  Why would I choose to install the plugin now, later, or never at all?  Does it have to do with what I plan to do with Flocker?  Or maybe the choice is based on my OS or Docker version or license constraints ...  This doesn't really tell me.